### PR TITLE
create test file and update proto

### PR DESCRIPTION
**Pull request by zinichakraborty**
I think there is missing implementation, but I will check with you guys in person on it tomorrow

### CODE REVIEW

**Comment by zinichakraborty** [comment]
There will be compiling errors - will only be able to get back to them after Thursday

File: packages/db-service/test/file_config.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    );
+  });
+
+  it('creates a new file service config', async () => {
+    const response: FileServiceConfig = await new Promise((resolve, reject) => {
+      configClient.createConfig(
+        {
+          config: {
+            projectId: '1',
+            buckets: FileServiceBucket[],
```
Comment (tmthecoder) [inline packages/db-service/test/file_config.e2e-spec.ts:63]: This should just be `[]`, no need for the `FileServiceBucket`

File: packages/proto/definitions/file_config.proto [hunk 2]
```diff
@@ -3,20 +3,39 @@ syntax = "proto3";
...
   rpc getConfig(GetFileServiceConfigRequest) returns (FileServiceConfig);
   rpc createConfig(CreateFileServiceConfigRequest) returns (FileServiceConfig);
   rpc deleteConfig(DeleteFileServiceConfigRequest) returns (FileServiceConfig);
   rpc updateConfig(UpdateFileServiceConfigRequest) returns (FileServiceConfig);
 }
 
-message FileServiceConfig {}
+message FileServiceConfig {
+  int64 id = 1;
+  repeated file_bucket.Bucket buckets = 2;
```
Comment (tmthecoder) [inline packages/proto/definitions/file_config.proto:19]: The build failure is coming from here, this (and all the others) should be `config.Bucket`

**Review by tmthecoder — CHANGES_REQUESTED** [review]
Make sure you run `yarn format:write` & `yarn lint` before pushing

**Review by tmthecoder — APPROVED** [review]
Made a few syntactic changes + controller/service issues that needed to be fixed

**Pull request merged**